### DISCUSSION TITLE
Installation: Unexplained changes when I push to Anvil host

### DISCUSSION THREAD

**Discussion by yahiakala** [anvilistas/anvil-extras#435] [Q&A] [answered]
I followed the instructions to install this app as a clone. I cloned it to my desktop and pushed it to an empty "Anvil Extras" app I have in my Anvil account. 

I force pushed the code to my Anvil Extras repo in Anvil and it all looked good.

With the first change I made in the code (I simply typed in something I deleted later), Anvil made a bunch of changes to the code including to these files:
- .anvil_editor.yaml
- anvil.yaml
- client_code/Autocomplete/form_template.yaml
- client_code/Chip/form_template.yaml
- client_code/ChipsInput/form_template.yaml
- client_code/MultiSelectDropDown/form_template.yaml
- client_code/Quill/form_template.yaml
- client_code/Slider/form_template.yaml
- client_code/Switch/form_template.yaml
- client_code/Tabs/form_template.yaml

Most of these changes look like reformatting yaml files:
<img width="618" alt="image" src="https://github.com/anvilistas/anvil-extras/assets/24422258/b97ea0f4-4a0c-4ead-919a-a4b3f806119e">

However one had a bunch of inserts:
<img width="618" alt="image" src="https://github.com/anvilistas/anvil-extras/assets/24422258/5824a780-c6c3-4e4c-9896-74d879ece91f">

I wanted to give a heads up as this may show up in my future pull requests, and I don't really have an explanation for it.

What should I do? Should I submit a pull request for this commit alone? 

Here's the full commit page: https://github.com/anvilistas/anvil-extras/compare/main...yahiakala:anvil-extras:yahia

**Comment by s-cork** [comment, marked as answer]
Thanks @meatballs and I don't always finish prs in the anvil editor. And the anvil editor will often fix up some yaml changes. In this case giving modules it knows about ids. Feel free to pr those changes.
